### DISCUSSION TITLE
TuringCourses/major/assemble/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#31] [General]
# TuringCourses/major/assemble/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major/assemble/

**Comment by ChangeFWorld** [comment]
不建议cs和ai的同学选这门课作为专业选修，知识和开发环境都十分古老，workload比较大，考试考些死记硬背，给分也不太行

**Comment by N0w13re** [comment]
如果cs专业后续模块课想选《汇编与接口》，那么这门课可以作为不错的过渡，因为《汇编与接口》里面一大半内容都在小白的汇编课中学过

**Comment by MTDickens** [comment]
**喜报**

23 级以及之后，信息安全专业不再需要必修这门垃圾课。而且小白不再教授密码学的课程，密码学的教案也从死记硬背+调库的白洪欢蜜汁工程密码学，变成了正经的密码学课程。自此，白洪欢正式退出 IS 必经的历史舞台。

**Reply by MTDickens** [reply]
source: 我的一个 IS 专业朋友
